Networking/NetworkingTests/Mapper/BlazeCampaignListMapperTests.swift

```diff
@@ -16,12 +16,12 @@ final class BlazeCampaignListMapperTests: XCTestCase {
         let item = try XCTUnwrap(campaigns.first)
         XCTAssertEqual(item.siteID, dummySiteID)
         XCTAssertEqual(item.campaignID, 34518)
-        XCTAssertEqual(item.productID, 134)
```

Comment:
nit: We should remove the `target_urn` field from the "blaze-campaigns-success" mock JSON file.

Reply:
Updated in a4647962ecc5c3a2ba47fd63fb523e75c4c0b011.